docs/running-workflows/supported-systems/serial/index.md

```diff
@@ -1,3 +1,6 @@
 # Serial
 
-A simple serial or sequential workflow system.
+The Serial workflow system is using the sequence workflow pattern where each
+computational step can run in a different containerised environment and the
+outputs of previous steps are passed along as inputs to later step. The next
```

Comment:
> outputs of previous steps are passed along as inputs to later step.

Perhaps we should use "later step**s**", in the same way we use the plural to relate to "previous steps", WDYT?